### ISSUE TITLE
question for inter-patient model

### ISSUE THREAD

**Issue by huinee56** [SajadMo/ECG-Heartbeat-Classification-seq2seq-model#16]
Could you please tell me how you train the classification model in the inter-patient mode? It is known that the training set is DS1 and the test set is DS2. Do you have a validation set? What are the specific details about how the validation set is obtained?
